data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
 			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
 		word
 			`"`
+
+news "unfettered hai dialog"
+	location
+		government "Hai (Unfettered)"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"How did you get here? You should've been blasted to bits by our ships."
+			"I came here because this is the only place in the galaxy where I feel wanted."
+			"Weak human. Why do you dare to come to the land of the true Hai?"
+			"Altered Hai are soft and weak-willed. They care not for true struggle or challenge."
+			"The last time a human called me 'cute', I ran him through with a machete."
```

Comment:
They probably never met a human on an Unfettered world before for this to be true...

Reply:
Tried fixing it, is it better?